modules/collection/programs/fzf.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}: let
+  inherit (lib.meta) getExe;
+  inherit (lib.modules) mkIf;
+  inherit (lib.options) mkEnableOption mkOption mkPackageOption;
+  inherit (lib.types) bool;
+
+  cfg = config.rum.programs.fzf;
+in {
+  options.rum.programs.fzf = {
+    enable = mkEnableOption "fzf";
+
+    package = mkPackageOption pkgs "fzf" {};
+
+    integrations = {
+      fish.enable = mkOption {
+        type = bool;
+        default = false;
+        example = true;
+        description = ''
+          Whether to enable fzf integration with fish.
+        '';
+      };
+      zsh.enable = mkOption {
+        type = bool;
+        default = false;
+        example = true;
+        description = ''
+          Whether to enable fzf integration with zsh.
+        '';
+      };
```

Always use mkEnableOption when possible.

```suggestion
      fish.enable = mkEnableOption "fzf integration with fish";
      zsh.enable = mkEnableOption "fzf integration with zsh";
```